integration_tests/integration/assess-recall.spec.js

```diff
@@ -29,14 +31,15 @@ context('Assess a recall', () => {
     cy.task('expectSearchResults', { expectedSearchTerm: nomsNumber, expectedSearchResults: searchResponse })
     cy.task('expectGetRecall', { recallId, expectedResult: { ...getRecallResponse, recallId } })
     cy.task('expectUpdateRecall', { recallId })
+    cy.task('expectPrisonList', { expectedResults: getPrisonList })
     cy.login()
     const recallsList = recallsListPage.verifyOnPage()
     recallsList.expectResultsCountText('1 recall')
     recallsList.results().find('tr').should('have.length', 1)
     const firstResult = recallsList.results().first()
     firstResult.get('[data-qa=name]').should('contain.text', 'Bobby Badger')
```

Comment:
use `personName` here?

Reply:
you've highlighted the line with the prison list mock, is that the line you meant to point to?